ui/address/AddressAccountHistory.tsx

```diff
@@ -0,0 +1,130 @@
+
+const getFilterValue = (getFilterValueFromQuery<NovesHistoryFilterValue>).bind(null, NovesHistoryFilterValues);
+
+type Props = {
+  scrollRef?: React.RefObject<HTMLDivElement>;
+}
+
+const AddressAccountHistory = ({ scrollRef }: Props) => {
+  const router = useRouter();
+
+  const currentAddress = getQueryParamString(router.query.hash).toLowerCase();
+
+  const [ filterValue, setFilterValue ] = React.useState<NovesHistoryFilterValue>(getFilterValue(router.query.filter));
+
+  const { data, isError, pagination, isPlaceholderData } = useQueryWithPages({
+    resourceName: 'noves_address_history',
+    pathParams: { address: currentAddress },
+    scrollRef,
+    options: {
+      placeholderData: generateListStub<'noves_address_history'>(NOVES_TRANSLATE, 10, { hasNextPage: false, pageNumber: 1, pageSize: 10 }),
+    },
+  });
+
+  const handleFilterChange = React.useCallback((val: string | Array<string>) => {
+
+    const newVal = getFilterValue(val);
+    setFilterValue(newVal);
+  }, [ ]);
+
+  const actionBar = (
+    <ActionBar mt={ -6 } pb={{ base: 6, md: 5 }}>
+      <AccountHistoryFilter
+        defaultFilter={ filterValue }
+        onFilterChange={ handleFilterChange }
+        isActive={ Boolean(filterValue) }
+        isLoading={ pagination.isLoading }
+      />
+
+      <Pagination ml={{ base: 'auto', lg: 8 }} { ...pagination }/>
+    </ActionBar>
+  );
+
+  const filteredData = isPlaceholderData ? data?.items : data?.items.filter(i => filterValue ? getFromToValue(i, currentAddress) === filterValue : i);
+
+  const content = (
+    <Box position="relative">
+      <Hide above="lg" ssr={ false }>
+        { filteredData?.map((item, i) => (
+          <AddressAccountHistoryListItem
+            key={ `${ i }-${ item.rawTransactionData.transactionHash }` }
+            tx={ item }
+            currentAddress={ currentAddress }
+            isPlaceholderData={ isPlaceholderData }
+          />
+        )) }
+      </Hide>
+
+      <Show above="lg" ssr={ false }>
+        <Table variant="simple" >
+          <TheadSticky top={ 75 }>
+            <Tr>
+              <Th width="120px">
+                  Age
+              </Th>
+              <Th>
+                  Action
+              </Th>
+              <Th width="320px">
+                  From/To
+              </Th>
+            </Tr>
+          </TheadSticky>
+          <Tbody maxWidth="full">
+            { filteredData?.map((item, i) => (
+              <AddressAccountHistoryTableItem
+                key={ `${ i }-${ item.rawTransactionData.transactionHash }` }
+                tx={ item }
+                currentAddress={ currentAddress }
+                isPlaceholderData={ isPlaceholderData }
+              />
+            )) }
+          </Tbody>
+        </Table>
+      </Show>
+    </Box>
+  );
+
+  return (
+    <>
+      { /* should stay before tabs to scroll up with pagination */ }
+      <Box ref={ scrollRef }></Box>
```

Comment:
Shouldn't it be passed as a prop like it's done for other tabs components on the address page?